Source/Core/Runtime/Configuration/BaseRuntimeConfiguration.cs

```diff
@@ -143,8 +150,44 @@ public virtual async Task<IProcess> LoadProcess(string path)
                     throw new ArgumentException("Given path is null or empty!");
                 }
 
-                byte[] serialized = await FileManager.Read(path);
-                return Serializer.ProcessFromByteArray(serialized);
+                int index = path.LastIndexOf("/");
+                string processFolder = path.Substring(0, index);
+                string manifestPath = $"{processFolder}/{ManifestFileName}.{Serializer.FileFormat}";
+
+                IProcessAssetManifest manifest;
+
+                if (await FileManager.Exists(manifestPath))
+                {
+                    byte[] manifestData = await FileManager.Read(manifestPath);
+                    manifest = Serializer.ManifestFromByteArray(manifestData);
+                }
+                else
+                {
+                    Debug.LogError("Manifest not found.");
+                    return null;
+                }
```

Comment:
I need a bit explanation on LoadProcess(....

- This is called when when we start the app in editor or on device, right?
- This LoadProcess is only working when the selected files folder contains a manifest, right? 
I can't find / understand how the old file loading works, is it contained in RuntimeConfigWrapper? 
- I think we should also refactor some logic out of this method to make it easier readable.

Reply:
Yes, that is called at runtime. The manifest not found looks like a nice bug. I have since changed things so that if you don't have a manifest it still loads the single file.
Agreed on refactoring.